backend/src/routers/gameRouter.ts

```diff
@@ -286,16 +302,22 @@ gameRouter.post('/filter', validateRequestBody(filterGamesSchema), async (req, r
  * ]
  *
  * @apiUse ZodError
- */
+ * @apiErrorExample {json} 401 Unauthorized:
+ * {
+ * 	"message": "Must be logged in to remove game"
+ * }
+*/
 gameRouter.post('/remove', async (req, res) => {
 	try {
-		await removeGame(req.body.id);
+		if (!req.user) { res.status(401).json({ message: 'Not logged in' }); return; }
```

Comment:
Documentation does not match actual return value

Reply:
Good catch :+1: